packages/redux/src/raffles/__tests__/reducer.test.ts

```diff
@@ -0,0 +1,240 @@
+import * as actionTypes from '../actionTypes';
+import * as fromReducer from './../reducer';
+import {
+  mockRaffleEstimationResponse,
+  mockRaffleParticipationResponse,
+  mockRaffleResponse,
+  mockRafflesResponse,
+  participationId_0,
+  participationId_1,
+  raffleId,
+} from 'tests/__fixtures__/raffles';
+import {
+  RaffleParticipation,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import reducer, { entitiesMapper } from '../reducer';
+import type { RaffleEntity } from '../../entities';
+import type { RafflesState } from '../types';
+
+let initialState: RafflesState;
+
+describe('raffles reducer', () => {
+  beforeEach(() => {
+    initialState = fromReducer.INITIAL_STATE;
+  });
+  describe('loading() reducer', () => {
```

Comment:
Missing test cases for the SUCCESS and ERROR actions. Please, review the order slices' reducers to make sure all cases are being tested. Also, when testing the success and error cases make sure to use a state which is not `undefined` so that the test properly catch bugs with the implementation.